salesagility/SuiteCRM
# Fix #10488 - Expand the number of filters in message queue views

## Description
This PR expands the number of filters available for the Email Queue module.

## Motivation and Context
https://github.com/salesagility/SuiteCRM/issues/10488

## How To Test This
1. Create a campaign with contacts, accounts, leads and users.
2. Send one or more marketing emails.
3. Access the message queue view and verify that the different filters added work correctly

## Types of changes


### Final checklist
